README.md

```diff
@@ -46,3 +46,5 @@ To see changes made to the `lookup` plugin in your admin UI, run (in the project
 
     $ cd -
     $ strapi build
+
+The plugin will be installed by `Dockerfile` already therefore you don't have to run these commands actually.
```

Comment:
That's only true for the former command (`npm install`), the second (`strapi build`) currently needs to run after changes to the plugin code during development (the docs recommend to run Strapi with `--watch-admin`, but I didn't get that to work with the Docker process). For this change here, I'd recommend we remove lines 40-43 in the README instead of adding the note in this line above.

Reply:
Thanks! I will remove the lines 40-43

Reply:
Regrading the `--watch-admin` option see PR https://github.com/hbz/strapi-rpb/pull/12